docs/operator-manual/declarative-setup.md

```diff
@@ -484,10 +484,13 @@ stringData:
   type: git
   url: https://github.com/argoproj/private-repo
   proxy: https://proxy-server-url:8888
+  noProxy: ".internal.example.com"
```

Comment:
A few places mention that this can be a list. Is it a CSV? If so, could we use a CSV example here?

Reply:
I updated the example, now it's list and contains different options.